Ultramules/BangazonApi
# Fixed all merge conflicts and fixed the EmployeeComputer table in SQL…

… to have UnAssignDate

# Pull Request---Ultra-Mules: Bangazon

### Issue Reference
https://github.com/Ultramules/BangazonApi/issues/30

### Description of Changes

- [ ] Clarity

- [ ] rationale of structure  

- [ ] Provide Context to paint mental picture 
I added the unassigndate property to the EmployeeComputer table to fix errors, I fixed all the merge conflicts that previously went unnoticed.
### Steps to Test
Fetch my branch and test your controllers and make sure all the information is what you had.
### System Configuration 

- [ ] 3rd Party Library 

- [ ] Command line utilities 

- [ ] UI Steps
Use Git Bash and Windows Powershell
### Will Review (Minimum: 2 Approvals)
Adelaide, Aaron
### Link to Ticket of Feature 
SEE ABOVE
